simplelru/list.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Probably this license should be added; can you copy over the LICENSE file and name it LICENSE_list and update the comment here to point to it? And at the top of the LICENSE_list file add a comment indicating it applies only to list.go?